mstp.c

```diff
@@ -3254,9 +3270,10 @@ static bool PTSM_run(port_t *prt, bool dry_run)
                 return false;
             if(prt->sendRSTP)
             { /* implement MSTP */
-                if(prt->newInfo || (prt->newInfoMsti && !mstiMasterPort)
+                if(!prt->dontTxmtBpdu && !prt->bpduFilterPort &&
```

Comment:
this also squeezes the `!prt->dontTxmtBpdu` condition in;
which didn't seem to be there before; and has no explanation why it's added